src/Page/Partners/Partner_.elm

```diff
@@ -113,10 +113,19 @@ view maybeUrl sharedModel static =
     }
 
 
+descriptionText : Data.PlaceCal.Partners.Partner -> String
+descriptionText partner =
+    if String.isEmpty partner.description then
+        "Please ask " ++ partner.name ++ " for more information"
```

Comment:
This needs to come from `Copy.Text`. Also not sure if where the copy came from - but does it need a fullstop?

All of the interface text is meant to be centralised there. You'll have to make a new key that takes a String as a param.

Let me know if you get stuck.